container/docker/factory.go

```diff
@@ -197,24 +197,38 @@ func Register(factory info.MachineInfoFactory, fsInfo fs.FsInfo, ignoreMetrics c
 	)
 
 	if dockerStorageDriver == devicemapperStorageDriver {
-		// If the storage drive is devicemapper, create and start a
-		// ThinPoolWatcher to monitor the size of container CoW layers with
-		// thin_ls.
-		dockerThinPoolName, err := dockerutil.DockerThinPoolName(*dockerInfo)
-		if err != nil {
-			return fmt.Errorf("couldn't find device mapper thin pool name: %v", err)
-		}
-
-		dockerMetadataDevice, err := dockerutil.DockerMetadataDevice(*dockerInfo)
-		if err != nil {
-			return fmt.Errorf("couldn't determine devicemapper metadata device")
+		_, err := devicemapper.ThinLsBinaryPresent()
```

Comment:
assume kubelet will not have to check this thing?  wondering if we should also list it as a soft/hard dependency there